integration/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "amazon-chime-sdk-js": "^3.0.0-beta.0",
```

Comment:
It would be good to update to `3.0.0-beta.1` which was released last Friday. But I can upgrade it in my PR after yours.

Reply:
Updated

Reply:
Thanks!

Reply:
I saw this dependency is still beta.0 version, could you update there as well?